.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,99 @@
+name: Prepare release
+
+env:
+  GIT_AUTHOR_EMAIL: "packages@datadoghq.com"
+  GIT_AUTHOR_NAME: "ci.datadog-api-spec"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: New version tag
+        required: true
+
+jobs:
+  prepare_release:
+    name: Create release PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get GitHub App token
+        id: get_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.PIPELINE_GITHUB_APP_ID }}
+          private_key: ${{ secrets.PIPELINE_GITHUB_APP_PRIVATE_KEY }}
+      - uses: actions/checkout@v2
+
+      - name: Setup Git
+        run: |
+          git config user.name "${GIT_AUTHOR_NAME}"
+          git config user.email "${GIT_AUTHOR_EMAIL}"
+
+      - name: Install Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: "8"
+          distribution: "adopt"
+
+      - name: Bump version
+        run: |
+          git switch -c "release/${RELEASE_TAG}"
+          mvn -DnewVersion=${RELEASE_TAG} versions:set
+          git commit -a -m "Version bump ${RELEASE_TAG}"
+          git push -f --set-upstream origin "release/${RELEASE_TAG}"
+        env:
+          RELEASE_TAG: ${{ github.event.inputs.tag }}
+
+      - name: Create PR
+        uses: actions/github-script@v5
+        env:
+          RELEASE_TAG: ${{ github.event.inputs.tag }}
+          BASE: ${{ github.event.ref }}
+        with:
+          github-token: ${{ steps.get_token.outputs.token }}
+          script: |
+            const { data: notes } = await github.rest.repos.generateReleaseNotes({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag_name: process.env.RELEASE_TAG,
+            });
+            const today = new Date().toJSON().slice(0, 10);
+            const header = [`# CHANGELOG\n\n## ${process.env.RELEASE_TAG} / ${today}\n`];
```

Comment:
strip v

Reply:
Yeah we shouldn't have v anywhere in java, so I don't think it applies.